include/emp/data/DataInterface.hpp

```diff
@@ -107,10 +107,10 @@ namespace emp {
     void PullData() { node->PullData(); }
 
     /// Reset this node. The exact effects of this depend on the modules that this node has,
-    /// but in general it prepares the node to recieve a new set of data.
+    /// but in general it prepares the node to receive a new set of data.
```

Comment:
wow you turned up so many typos :spider: !!!

not for this pr, but makes me wonder if there's an easy way to lint comments for grammar/spelling as part of CI

Reply:
I installed a spellchecker extension in VSCode. It's so great! I don't know if it would be easy to automate, though - tons of false positives